### PR TITLE
feat(js): add llama-3.2 support to model garden plugin

### PR DESCRIPTION
**Pull request by huangjeff5**
Checklist (if applicable):
- [ x ] Tested (manually, unit tested, etc.)

### CODE REVIEW

File: js/plugins/vertexai/src/model_garden.ts [hunk 1]
```diff
@@ -29,6 +29,23 @@ export const ModelGardenModelConfigSchema = OpenAIConfigSchema.extend({
   location: z.string().optional(),
 });
 
+export const llama32 = modelRef({
+  name: 'vertexai/llama-3.2',
+  info: {
+    label: 'Llama 3.2',
+    supports: {
+      multiturn: true,
+      tools: true,
```
Comment (pavelgj) [inline js/plugins/vertexai/src/model_garden.ts:38]: Are tools working?

**Review by pavelgj — APPROVED** [review]

**Comment by pavelgj** [comment]
Could you please make the same change to the next branch?

File: js/plugins/vertexai/src/model_garden.ts [hunk 2]
```diff
@@ -29,6 +29,23 @@ export const ModelGardenModelConfigSchema = OpenAIConfigSchema.extend({
...
+      media: true,
+      systemRole: true,
+      output: ['text', 'json'],
+    },
+    versions: ['meta/llama-3.2-11b-vision-instruct-maas'],
+  },
+  configSchema: ModelGardenModelConfigSchema,
+  version: 'meta/llama-3.2-11b-vision-instruct-maas',
```
Comment (pavelgj) [inline js/plugins/vertexai/src/model_garden.ts:46]: could you please double check, I believe the maas model is 90b (`meta/llama-3.2-90b-vision-instruct-maas`)

File: js/plugins/vertexai/src/model_garden.ts (hunk 1 above)
Comment (huangjeff5) [reply js/plugins/vertexai/src/model_garden.ts:38]: Wasn't able to test on Dev UI, did not test manually but I believe it should be supported in the model. Let me double check.

File: js/plugins/vertexai/src/model_garden.ts (hunk 2 above)
Comment (huangjeff5) [reply js/plugins/vertexai/src/model_garden.ts]: Good catch

**Comment by pavelgj** [comment]
coud you please also update https://github.com/firebase/genkit/blob/main/docs/plugins/vertex-ai.md#llama-31-405b-on-vertex-ai-model-garden

**Pull request merged**